idd/Energy+.idd.in

```diff
@@ -44351,9 +44351,7 @@ ZoneHVAC:LowTemperatureRadiant:ConstantFlow:Design,
         \min-fields 11
    A1 , \field Name
         \required-field
-        \reference-class-name validBranchEquipmentTypes
-        \reference validBranchEquipmentNames
-        \reference ZoneEquipmentNames
+        \reference RadiantDesignObject
```

Comment:
Wrong reference. OpenStudio enforces this more strictly and I had a failing test because of it

Reply:
A great fix, thanks @jmarrec